src/style.css

```
@@ -1 +1,33 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  height: 16px;
+  margin-bottom: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-size: cover;
+  background-image: url(./images/star.svg);
```

```suggestion
  background-size: cover;
  background-image: url(./images/star.svg);
  background-position: 50%;
```
add this property here